src/JWT/Serializers/SystemTextJsonSerializer.cs

```diff
@@ -0,0 +1,60 @@
+﻿#if SYSTEMTEXTJSON
+namespace JWT.Serializers
+{
+    using System;
```

Comment:
usings outside the namespace, please

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1200.md